packages/core/admin/ee/server/src/controllers/index.ts

```diff
@@ -3,13 +3,11 @@ import type {} from 'koa-body';
 import authentication from './authentication';
 import role from './role';
 import user from './user';
-import auditLogs from './audit-logs';
 import admin from './admin';
 
 export default {
   authentication,
   role,
   user,
-  auditLogs,
```

Comment:
what's replacing this now?

Reply:
I add the controllers, routes etc... here `packages/core/admin/ee/server/src/index.ts` in one place to avoid doing the check for the licens/feature enabled in multiple files.